server/main.lua

```diff
@@ -1,6 +1,12 @@
 assert(lib.checkDependency('qbx_core', '1.15.1', true))
 assert(lib.checkDependency('qbx_vehicles', '1.3.1', true))
 
+local triggerEventHooks = lib.load('@qbx_core.modules.hooks')
+
+function GaragesHooks(...) -- anti-dubliquate, avoid deleting the parkVehicle hook and use it in 'spawn-vehicle.lua'.
+    return triggerEventHooks(...)
+end
```

Comment:
Just load the hooks module in the other file it's being used in as well. I don't see a benefit to wrapping it here in a global.